examples/main/java/io/milvus/resourcegroup/ResourceGroupManagement.java

```diff
@@ -0,0 +1,281 @@
+     * resource group to 0 first.
+     * 
+     * @param resourceGroupName
+     */
+    public void dropResourceGroup(String resourceGroupName) throws Exception {
+        R<RpcStatus> resp = client
+                .dropResourceGroup(DropResourceGroupParam.newBuilder().withGroupName(resourceGroupName).build());
+        unwrap(resp);
+    }
+
+    /**
+     * Scale to the number of nodes in a resource group.
+     * 
+     * @param resourceGroupName
+     * @param requestNodeNum
+     */
+    public void scaleResourceGroupTo(String resourceGroupName, Integer requestNodeNum) throws Exception {
+        if (resourceGroupName == RECYCLE_RG) {
+            throw new IllegalArgumentException("Cannot scale to recycle resource group");
+        }
+        // Update a resource group with given nodeNum.
+        // Set the resource group transfer to recycle resource group by default.
+        R<RpcStatus> resp = client
+                .updateResourceGroups(UpdateResourceGroupsParam.newBuilder()
+                        .putResourceGroup(resourceGroupName, ResourceGroupConfig.newBuilder()
+                                .withRequests(new ResourceGroupLimit(requestNodeNum))
+                                .withLimits(new ResourceGroupLimit(requestNodeNum))
+                                .withFrom(Arrays.asList(new ResourceGroupTransfer(RECYCLE_RG)))
+                                .withTo(Arrays.asList(new ResourceGroupTransfer(RECYCLE_RG)))
+                                .build())
+                        .build());
+        unwrap(resp);
+    }
+
+    /**
+     * Transfer a database to specified resource group.
+     * Only support single replica now.
+     * 
+     * @param dbName            The name of the database to transfer.
+     * @param resourceGroupName The name of the target resource group.
+     */
+    public void transferDataBaseToResourceGroup(String dbName, String resourceGroupName) throws Exception {
+        R<ShowCollectionsResponse> resp = client.showCollections(ShowCollectionsParam.newBuilder()
+                .withDatabaseName(dbName)
+                .build());
+        ShowCollectionsResponse showCollectionResponse = unwrap(resp);
+
+        for (String collection : showCollectionResponse.getCollectionNamesList()) {
+            String currentResourceGroupName = getCollectionResourceGroupName(dbName, collection);
+            // skip if the collection is not loaded or is already added to resourceGroup.
+            if (currentResourceGroupName == null || currentResourceGroupName == resourceGroupName) {
+                continue;
+            }
+            R<RpcStatus> status = client.transferReplica(TransferReplicaParam.newBuilder()
+                    .withDatabaseName(dbName)
+                    .withCollectionName(collection)
+                    .withSourceGroupName(currentResourceGroupName)
+                    .withTargetGroupName(resourceGroupName)
+                    .withReplicaNumber(1L)
```

Comment:
why just we transfer only one replica to target resource group?

Reply:
It's an example. For easy coding, add a single replica limits here.
I will add a comment to explain it.